hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot

```diff
@@ -21,32 +21,8 @@ ${OZONE_S3_HEADER_VERSION}     v4
 ${OZONE_S3_SET_CREDENTIALS}    true
 
 *** Keywords ***
-Execute AWSS3APICli
```

Comment:
Not clear why these keywords are moved to the commonlib. Is there any reason for that?

Reply:
This can be used inside security robot tests as well.